meshtastic/interfaces/ble/lifecycle_compat_service.py

```
@@ -0,0 +1,1582 @@
+        Returns
+        -------
+        None
+            Always returns ``None``.
+
+        Raises
+        ------
+        Exception
+            Propagates thread-start failures after clearing stale thread state.
+        """
+        coordinator = BLELifecycleService._receive_lifecycle_coordinator(iface)
+        start_receive_thread = getattr(coordinator, "start_receive_thread")
```

Ruff (0.15.6)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)